vuepress/docs/v7.0/tutorials/create/pb/publish-simple-bundle.md

```diff
@@ -4,138 +4,146 @@ sidebarDepth: 2
-``` sh
-git init
-git add .
-git commit -m "Init Git repository"
-```
-
-3. Add your remote repository as origin and push the bundle
-``` sh
-git remote add origin https://your/remote/repository.git
-git push -u origin master
-```
+1. From the bundle directory, run the following commands to initialize Git and commit the files:
+   ``` sh
+   git init
+   git add .
+   git commit -m "Init Git repository"
+   ```
+
+2. Add your remote repository as origin and push the bundle:
+   ``` sh
+   git remote add origin https://your/remote/repository.git
+   git push -u origin master
+   ```
+
+3. Publish a Git tag:
+   ``` sh
+   git tag -a "v0.0.1" -m "My first tag"
+   git push --tags
+   ```
+
+5. Create the Kubernetes custom resource for your bundle: 
+   ``` sh
+   ent bundler from-git --name=example-bundle --namespace=entando --repository=https://YOUR-REMOTE-REPOSITORY.git --dry-run > example-bundle.yaml
+   ```
+   A bundle must be published to Git before you can create a custom resource for it. Provide the URL of the bundle's remote Git repository (ends with ".git"). To include a thumbnail for your bundle, use the option  `--thumbnail-file` or `--thumbnail-url`.
+
+6. Transfer the file to your VM, e.g:
+   ```
+   multipass transfer example-bundle.yaml entando:
+   ```
+
+7. Apply the bundle definition to Kubernetes:
+   ```
+   kubectl -n entando apply -f example-bundle.yaml
+   ```
+
+8. Confirm the presence of your custom resource:
+   ```
+   kubectl get EntandoDeBundle -n entando
+   ```
 
-4. Publish a Git tag
-``` sh
-git tag -a "v0.0.1" -m "My first tag"
-git push --tags
-```
-
-5. Now that you've published your bundle to Git, you can create the Kubernetes custom resource for it.
-
- Run the `ent bundler from-git` command, providing your remote Git repository URL via the `--repository` option and the correct namespace via `--namespace`. You can also include a thumbnail for your bundle with `--thumbnail-file` or `--thumbnail-url`.
-
-``` sh
-ent bundler from-git --name=example-bundle --namespace=entando --repository=https://your/remote/repository.git --dry-run > example-bundle.yaml
-```
-
-6. Apply this definition to Kubernetes. You may need to first transfer the file to your VM, e.g with `multipass transfer`.
+## Install the Bundle into an Application
 
-```
-kubectl -n entando apply -f example-bundle.yaml
-```
+1. In your Entando instance, go to `App Builder` → `Component Repository` 
 
-7. Confirm the presence of your custom resource with the command `kubectl get EntandoDeBundle -n entando`.
+2. Click `Install`. A bundle with multiple iterations allows version selection.
 
-## Install the Bundle into an Application
-Your bundle should appear in `App Builder` → `Component Repository` in your Entando instance. Clicking `Install` should allow version selection if your bundle has multiple iterations.
+3. Verify the `Install` button changes to `Uninstall` to indicate that the installation completed successfully
```

Comment:
simplify >> the installation is successful when the button reads Uninstall

Reply:
each step should be an instruction, telling the user to do something